pages/visualizations/overview/issues_over_time.py

```diff
@@ -106,33 +98,74 @@ def toggle_popover_3(n, is_open):
     return is_open
 
 
+columns = ["1", "2", "3"]
```

Comment:
what is this? seems like it shouldnt be here

Reply:
yeah that's an old artifact, I'll change that.